cmd/otelcorecol/go.mod

```diff
@@ -59,6 +59,11 @@ require (
 	go.opentelemetry.io/collector/semconv v0.56.0 // indirect
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.34.0 // indirect
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.34.0 // indirect
+	go.opentelemetry.io/contrib/propagators/autoprop v0.34.0 // indirect
```

Comment:
Worried about this dependency, since may get quite big. Can we limit to a subset of propagators for the moment?

Reply:
Good concern to have, I think b3 and traceparent are the most used ones. Let me see if I can compile without getting other ones.

Reply:
Waiting on this?

Reply:
@bogdandrutu the way to limit the propagtors is to not use autoprop module, the module  - if we go about using go-contrib/propagators directly then we can limit - The propagators can be enabled via configuration strings but the logic will become sort of parallel/redundant  to what autoprop is doing but with limited options. Are you ok with this ?

Reply:
Done!